assets/sass/components/_heroes.scss

```diff
@@ -160,6 +159,68 @@
     }
 }
 
+.homepage-banner {
```

Comment:
Duplicated these styles for now to account for the banner being separated from the image.